README.md

```diff
@@ -62,6 +62,20 @@ type Config struct {
 }
 ```
 
+### Backend tag
+
+As we already said, Confita will call each backend one after another. But in order to avoid some useless processing we can specify in which backend the value of a field can be found with the `backend` tag:
+
+```go
+type Config struct {
+  Host        string        `config:"host" backend:"env"`
```

Comment:
:grey_question: Should we keep only one tag or two ?

```go
Host string `config:"host" backend="env"`
```
```go
Host string `config:"host,backend=env"`
```

With the `required` option:

```go
Host string `config:"host,required" backend="env"`
```

```go
Host string `config:"host,required,backend=env"`
```

cc @jhchabran

Reply:
I prefer the second one because we avoid polluting the struct with multiple tags, we have our `config` namespace and use options. Also we avoid having a weird separation when using the `required` option.

Reply:
I agree, you're right and it'd be probably simpler to remember it that way (i.e. no special case).